packages/tldraw/src/translations/fr.json

```diff
@@ -3,96 +3,96 @@
+  "redo": "Rétablir",
   "cut": "Couper",
   "copy": "Copier",
   "paste": "Coller",
-  "copy.as": "Copier en tant que",
-  "export.as": "Exporter en tant que",
-  "select.all": "Sélectionner tout",
-  "select.none": "Sélectionner aucun",
+  "copy.as": "Copier sous",
+  "export.as": "Exporter sous",
+  "select.all": "Tout sélectionner",
+  "select.none": "Tout désélectionner",
   "delete": "Supprimer",
-  "new.project": "Nouveau Project",
+  "new.project": "Nouveau project",
   "open": "Ouvrir",
   "save": "Enregistrer",
-  "save.as": "Enregistrer en tant que",
-  "upload.media": "Uploader Un Média",
-  "create.page": "Créer une Page",
+  "save.as": "Enregistrer sous",
+  "upload.media": "Importer un média",
+  "create.page": "Créer une page",
   "new.page": "Nouvelle Page",
-  "page.name": "Nom de la Page",
+  "page.name": "Nom de la page",
   "duplicate": "Dupliquer",
   "cancel": "Annuler",
   "copy.invite.link": "Copier le lien d'invitation",
   "copy.readonly.link": "Copier le lien en lecture seule d'invitation",
-  "create.multiplayer.project": "Créer un Project Multi-joueurs",
-  "copy.multiplayer.project": "Copier dans un Projet Multi-joueurs",
-  "select": "Selection",
+  "create.multiplayer.project": "Créer un project multi-joueurs",
+  "copy.multiplayer.project": "Copier dans un projet multi-joueurs",
+  "select": "Sélection",
   "eraser": "Gomme",
   "draw": "Crayon",
   "arrow": "Flèche",
   "text": "Text",
-  "sticky": "Papier collant",
+  "sticky": "Note",
   "Rectangle": "Rectangle",
   "Ellipse": "Cercle",
   "Triangle": "Triangle",
   "Line": "Ligne",
-  "rotate": "Retourner",
-  "lock.aspect.ratio": "Verouiller l'Aspect Ratio",
-  "unlock.aspect.ratio": "Déverouiller l'Aspect Ratio",
+  "rotate": "Rotation",
+  "lock.aspect.ratio": "Verrouiller les proportions",
+  "unlock.aspect.ratio": "Déverrouiller les proportions",
   "group": "Grouper",
   "ungroup": "Dégrouper",
-  "move.to.back": "Envoyer vers l'arrière",
-  "move.backward": "Mettre en arrière-plan",
-  "move.forward": "Mettre au premier plan",
-  "move.to.front": "Envoyer vers l'avant",
-  "reset.angle": "Réinitialiser l'Angle",
-  "lock": "Verouiller",
-  "unlock": "Déverouiller",
+  "move.to.back": "Placer à l'arrière plan",
+  "move.backward": "Reculer d'un plan",
+  "move.forward": "Avancer d'un plan",
+  "move.to.front": "Placer au premier plan",
+  "reset.angle": "Réinitialiser la rotation",
+  "lock": "Verrouiller",
+  "unlock": "Déverrouiller",
   "move.to.page": "Déplacer vers la page",
-  "flip.horizontal": "Retourner Horizontalement",
-  "flip.vertical": "Retourner Verticalement",
-  "move": "Mettre",
-  "to.front": "À l'avant",
-  "forward": "Au premier plan",
-  "backward": "En arrière plan",
-  "back": "À l'arrière",
+  "flip.horizontal": "Retourner horizontalement",
+  "flip.vertical": "Retourner verticalement",
+  "move": "Disposition",
+  "to.front": "Au premier plan",
+  "forward": "Avancer",
+  "backward": "Reculer",
+  "back": "En arrière plan",
```

Comment:
I'm not sure if these are the right words, 
Mettre à l'avant, Mettre à l'arrière
Mettre au premier plan
Mettre en arrière plan
I think that is the right word, IMO.

Reply:
you can see how other tools like Excalidraw or figma is translating these words

Reply:
Hi @judicaelandria,

Thanks for the feedback, good observation.
I will try to explain why it is not correct for a drawing app.

**Figma** is a bad reference, because it uses Google translate... see the doc [link](https://help.figma.com/hc/en-us/articles/4404356228119-Change-Figma-s-language-using-Google-Chrome).
For simple words, it does the job. But there is a cruel lack of consistency with too many translation errors.
As a funny (or not) example: it translates "**stroke**" into "**cerebral vascular accident**" 😵‍💫.
- In our case, the result is:

| English | French (Google) | Comment |
|-----------|-----------|-----------|
| Bring to front | Mettre au premier plan | Why not. |
| Send to back | Envoyer au fond | It make no sense for a drawing app in French.<br />literally: Send at the bottom.<br />Like throwing a stone at the bottom of the sea. |

**Excalidraw** use:
| English | French | Comment |
|-----------|-----------|-----------|
| Send backward | Envoyer vers l'arrière | As Figma, the verb **Envoyer** is not appropriate.<br /> This does not allow to understand the action of this function.  |
| Bring forward | Envoyer vers l'avant | As Figma, the verb **Envoyer** is not appropriate.<br /> This does not allow to understand the action of this function.  |
| Send to back | Mettre en arrière plan | Why not. |
| Bring to front | Mettre au premier plan | Why not. |

**tldraw** now:
| English | French (Before the PR) | Comment |
|-----------|-----------|-----------|
| Move > To Front | Mettre > À l'avant | The translation was inverted with that of "Move / Forward".  |
| Move > Forward | Mettre > Au premier plan | The translation was inverted with that of "Move / To Front". |
| Move > Backward | Mettre > En arrière plan | The translation was inverted with that of "Move / To Back". |
| Move > To Back | Mettre > À l'arrière | The translation was inverted with that of "Move / Backward". |

In tldraw, when you open the context menu, you will first see the **Move** action in English.
This is coherent when we want to change the order of the position of an object.
Because we have the information of a moving action.

But in French, the verb **Mettre** is the action of putting something/someone somewhere.
Display like this, alone in the context menu, it means nothing.

I commented "_Why not_" for "_Mettre au premier plan_".
Because in **one sentence only**, we understand the goal of the function.
Even if the verb **Mettre** is not the best choice.
Would you replace the verb **move** with **put** in the English version?

For "Mettre à l'avant" and "Mettre à l'arrière" even in one sentence, it's not appropriate.

My choice to give the right context information at the first level of the contextual menu.
With "Disposition", it's exactly the scope of the functionality.  
Then, in the sub-menu the dedicated action verbs that describe the right move.

| English | French (Last commit) | Comment |
|-----------|-----------|-----------|
| Move > To Front | Disposition > Déplacer au premier plan | "Déplacer au premier plan" translates to "Move to front". |
| Move > Forward | Disposition > Avancer | "Avancer" translates to "Move forward". |
| Move > Backward | Disposition > Reculer  | "Reculer" translates to "Move backward". |
| Move > To Back | Disposition > Déplacer à l'arrière plan  | "Déplacer  à l'arrière plan" translates to "Move to back". |

What do you think?